runner.py

```diff
@@ -22,6 +22,7 @@
 BOT_CONFIG_LOADOUT_HEADER = 'Participant Loadout'
 BOT_CONFIG_LOADOUT_ORANGE_HEADER = 'Participant Loadout Orange'
 BOT_CONFIG_MODULE_HEADER = 'Bot Location'
+BOT_CONFIG_AGENT_HEADER = 'Agent Configuration'
```

Comment:
We'd be calling it three different things in the same file, which is confusing - Participant, Bot, and now Agent. I don't think there's any meaningful difference, so I'd suggest calling this "Bot Arguments" or "Bot Parameters"